keyboards/zinc/keymaps/toshi0383/keymap.c

```diff
@@ -0,0 +1,147 @@
+   * |      |   1  |   2  |   3  |  4   |   5  |             |   6  |   7  |   8  |   9  |   0  |      |
+   * |------+------+------+------+------+------|             |------+------+------+------+------+------|
+   * |      |   :  |   ;  |  "   |  '   |  \   |             | Left | Down | Up   |Right |      |      |
+   * |------+------+------+------+------+------|             |------+------+------+------+------+------|
+   * |      |      |      |      |      |      |             |ADJUST|      |      |      |      |      |
+   * |------+------+------+------+------+------|             |------+------+------+------+------+------|
+   * |      |      |      |      | GUI  |Shift |             |Shift |------|      |      |      |      |
+   * `-----------------------------------------'             `-----------------------------------------'
+   */
+  [_RAISE] = LAYOUT_ortho_4x12( \
+  _______, KC_1,    KC_2,    KC_3,    KC_4,    KC_5,           KC_6,    KC_7,    KC_8,    KC_9,    KC_0,  _______, \
+  _______, KC_COLN, KC_SCLN, KC_DQT, KC_QUOT, KC_BSLS,         KC_LEFT, KC_DOWN, KC_UP, KC_RGHT, _______, _______, \
+  _______, _______, _______, _______, _______, _______,        ADJUST, _______, _______, _______, _______, _______, \
+  _______, _______, _______, _______, KC_LGUI, KC_LSFT,        KC_LSFT, _______, _______, _______, _______, _______ \
+  ),
+
+  /* Adjust (Lower + Raise)
+   * ,-----------------------------------------.             ,-----------------------------------------.
+   * |      |      |      |      |      |      |             |      |      |      |      |      |      |
+   * |------+------+------+------+------+------|             |------+------+------+------+------+------|
+   * |      |      |      |      |      |      |             |      |      |      |      |      |      |
+   * |------+------+------+------+------+------|             |------+------+------+------+------+------|
+   * |      |      |      |      |      |      |             |      |      |      |      |      |      |
+   * |------+------+------+------+------+------|             |------+------+------+------+------+------|
+   * |      |      |      |      |      |RESET |             |------|      |      |      |      |      |
+   * `-----------------------------------------'             `-----------------------------------------'
+   */
+    [_ADJUST] =  LAYOUT_ortho_4x12( \
+      _______, _______, _______, _______, _______, _______,  _______, _______, _______, _______, _______, _______, \
+      _______, _______, _______, _______, _______, _______,  _______, _______, _______, _______, _______, _______, \
+      _______, _______, _______, _______, _______, _______,  _______, _______, _______, _______, _______, _______, \
+      _______, _______, _______, _______, _______, RESET,  _______, _______, _______, _______, _______, _______ \
+      ),
+
+};
+
+// define variables for reactive RGB
+bool TOG_STATUS = false;
```

Comment:
Wait, what are you trying to do there?

Reply:
Oh thank you that line is unnecessary too.
I copied the default keymap and removed LED related code.

Reply:
I’ll remove this line!

And I also have issue not able to enter ADJUST layer by RAISE + N.
Could someone please help debugging?

Reply:
Oh the issue is on reviung39 keymap, not zinc.

Reply:
Di you still want to clean this up?

Reply:
Done in https://github.com/qmk/qmk_firmware/pull/7141/commits/8791b3d4a6db79bdd7779ea62984d59451ec665c